src/compiler/parser.rs

```diff
@@ -139,7 +140,7 @@ pub fn create_source_file(file_name: &str, source_text: &str) -> SourceFile {
     Parser().parse_source_file(file_name, source_text)
 }
 
-enum MissingNode {
+pub enum MissingNode {
```

Comment:
Here the macro's assumption that its top-level enum is in `crate::` scope was being violated